internal/gqlclient/models.go

```diff
@@ -189,8 +189,9 @@ type MutableCodeChangeSource struct {
 }
 
 type CreateCodeChangeSourceMutationInput struct {
-	ProjectSlug       string `json:"projectSlug"`
-	InitializeChanges bool   `json:"initializeChanges"`
+	ProjectSlug       		string `json:"projectSlug"`
+	InitializeChanges 		bool   `json:"initializeChanges"`
+	InitializeChangesNow 	bool   `json:"initializeChangesNow"`
```

Comment:
This is formated a bit weird, probably `make format` will get this in order

Reply:
Uh, I only see it now in diff (tabs vs spaces 😊), will fix, tnx!

Reply:
`gofmt` to the rescue :D